src/BloomExe/Edit/EditingView.cs

```diff
@@ -711,6 +717,7 @@ private void OnChangeImage(DomEventArgs ge)
 				try
 				{
 					imageInfo = PalasoImage.FromFile(existingImagePath);
+					_originalSize = imageInfo.Image.PhysicalDimension;
```

Comment:
We had a user whose images for a book all had exactly the same name.  This strikes me as bizarre, but it was a real user reporting that the program was behaving strangely.

I think copyrights are handled okay the way they're working right now, but I probably should double check.  (I remember testing this, but my memory sometimes remembers things that didn't happen.)  A future tool to change contrast, color scheme, etc. would be problematic.